x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules/impact_hosts_file_modified.json

```diff
@@ -58,5 +58,5 @@
   "timeline_title": "Comprehensive File Timeline",
   "timestamp_override": "event.ingested",
   "type": "eql",
-  "version": 7
+  "version": 8
```

Comment:
This was already version bumped in the last `8.2 Rules` PR: https://github.com/elastic/kibana/pull/129526/files#diff-ce969c996a1c400debb065fec595e7d3ff5235fc151eeefa853c58963b02e45fR61